tests/unit/test-class-wc-payments-payment-request-button-handler.php

```diff
@@ -150,7 +158,8 @@ private function make_wcpay_gateway() {
 			$mock_action_scheduler_service,
 			$mock_rate_limiter,
 			$mock_order_service,
-			$mock_dpps
+			$mock_dpps,
+			$this->mock_localization_service
```

Comment:
Here we don't need a new property, and we can dispatch `$this->createMock( WC_Payments_Localization_Service::class )` right here.